.ci/msys_test.bat

```diff
@@ -0,0 +1,7 @@
+:: FIXME(equalsraf) currently not all tests work on Windows
+:: for now just run what we can
+mingw32-make functionaltest TEST_FILE=test\functional\eval\printf_spec.lua VERBOSE=1 || goto :error
```

Comment:
Should we use a test tag for this? That would enable whatever granularity we want, and would put the information what is supported directly at the tests.